src/less/chat.less

```diff
@@ -10,7 +10,14 @@
   font-size: 13px;
   position: relative;
 }
-
+#chat-log .message.whisper {
+  background: #e8e8ef;
```

Comment:
This needs to be `background-color` instead of just `background`. We use a transparent `.webp` image on the background, so any color you set will show through the image. Using `background` is overriding that image, and we need to keep the image to maintain consistency with the rest of the design.

Also, the chosen color doesn't really work in this context. I realize you just copied the hex code being used in foundry core, but it doesn't stand out against the Starfinder color scheme very well. It's a little too subtle. We need something that's going to say "I'm obviously different form the other chat cards".

Reply:
Looking at the blue for the header (https://colour-palette-generator.azurewebsites.net/?r=150&g=200&b=255), we can definitely find some complementary colors to that. For this color being a gray, there's not a good complementary gray, but https://colour-palette-generator.azurewebsites.net/?r=112&g=118&b=191 might work, and should look good with the border below.